src/router/routes.ts

```diff
@@ -1,63 +1,39 @@
-import { Signin, Signout, Landing } from '@/views'
+import { Landing, Signin, Signout } from '@/views'
+import { Routes } from '@/enums'
 
 export const routes = [
   {
     path: '/',
-    name: 'landing', // *** TODO: make this "Request a Business Name" tab
+    name: Routes.REQUEST, // "Request a Business Name" tab
+    component: Landing
+  },
+  {
+    path: '/manage',
```

Comment:
Future route... I'm working on it... Only moderate success so far but basically this displays the Manage tab while "/" displays the Request tab.